test/cypress/integration/simplified-account/service-settings/card-payments/card-payments.cy.js

```diff
@@ -0,0 +1,157 @@
+const userStubs = require('@test/cypress/stubs/user-stubs')
+const gatewayAccountStubs = require('@test/cypress/stubs/gateway-account-stubs')
+const ROLES = require('@test/fixtures/roles.fixtures')
+
+const USER_EXTERNAL_ID = 'user-123-abc'
+const SERVICE_EXTERNAL_ID = 'service-456-def'
+const GATEWAY_ACCOUNT_ID = 11
+const ACCOUNT_TYPE = 'test'
+
+const pageUrl = `/simplified/service/${SERVICE_EXTERNAL_ID}/account/${ACCOUNT_TYPE}/settings/card-payments`
+
+const setupStubs = ({
+  role,
+  collectBillingAddress,
+  defaultBillingAddressCountry,
+  allowApplePay,
+  allowGooglePay,
+  serviceName
+} = {}) => {
+  cy.task('setupStubs', [
+    userStubs.getUserSuccess({
+      userExternalId: USER_EXTERNAL_ID,
+      gatewayAccountId: GATEWAY_ACCOUNT_ID,
+      serviceName: { en: serviceName ?? 'My card payment service' },
+      serviceExternalId: SERVICE_EXTERNAL_ID,
+      role: ROLES[role ?? 'admin'],
+      collectBillingAddress: collectBillingAddress ?? true,
+      defaultBillingAddressCountry: defaultBillingAddressCountry ?? 'GB',
+      features: 'degatewayaccountification' // TODO remove features once simplified accounts are live
+    }),
+    gatewayAccountStubs.getAccountByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, {
+      gateway_account_id: GATEWAY_ACCOUNT_ID,
+      allow_apple_pay: allowApplePay ?? true,
+      allow_google_pay: allowGooglePay ?? true
+    })
+  ])
+}
+
+describe('Card payments page', () => {
+  beforeEach(() => {
+    cy.task('clearStubs')
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+  })
+
+  describe('Card payments landing page', () => {
+    describe('for an admin user', () => {
+      it('should show the correct heading and title', () => {
+        setupStubs()
+        cy.visit(pageUrl)
+
+        cy.get('h1').should('contain.text', 'Card payments')
+        cy.title().should('eq', 'Settings - Card payments - GOV.UK Pay')
+      })
+
+      it('should display the provided card payment details (version 1 - everything on)', () => {
```

Comment:
Can this be in a `describe('when all settings are on')`? similarly for when all settings are off

Reply:
Yeah, I kept switching between different options, the problem is that one of them is a country code so 'everything on' isn't a great description but when I was getting confused between the tests I added it to avoid confusion.  I have considered using `should display the default details` and `should display user-set details` but that implies that the test would be updated in line with any changes to the defaults and I doubt that would happen.